libvirt/resource_libvirt_domain.go

```diff
@@ -158,6 +158,10 @@ func resourceLibvirtDomain() *schema.Resource {
 							Type:     schema.TypeString,
 							Optional: true,
 						},
+						"cache": {
```

Comment:
Add default value here

Reply:
As i understand it, a default value is not needed here. The default cache mode from the underlying components will be used if the setting is not available in the domains disk xml:

"If no cache mode is specified, an appropriate default cache mode is used. "

that mode may be chosen by qemu then.

Reply:
OK I will look later thx for clarification, was on mobile phone